### ISSUE TITLE
Error message arrows are visually offset when running a source file that indents with tabs

### ISSUE THREAD

**Issue by webbedspace** [nushell/nushell#6742]
### Describe the bug

See title.

### How to reproduce

Create the file `temp.nu` containing this (notice the use of tabs - excessive use is simply to make the effect obvious):
```
def foo [] {
				cd "A:/Games/Auditorium" && Auditorium.exe
}
```
The error message resulting from running `source temp.nu` is as follows:
![image](https://user-images.githubusercontent.com/83939/195979153-1636bb18-7853-4b0e-b8da-a33ab917336a.png)

### Expected behavior

As you can see, the arrow does not point at the `&&` symbol even though that's the expected behaviour.

### Screenshots

See above.

### Configuration

| key                | value                                           |
| ------------------ | ----------------------------------------------- |
| version            | 0.69.1                                          |
| branch             |                                                 |
| commit_hash        | 8a5abc7afc28a36908a32f1852127e2a5170b461        |
| build_os           | windows-x86_64                                  |
| build_target       | x86_64-pc-windows-msvc                          |
| rust_version       | rustc 1.64.0 (a55dd71d5 2022-09-19)             |
| rust_channel       | stable-x86_64-pc-windows-msvc                   |
| cargo_version      | cargo 1.64.0 (387270bc7 2022-09-16)             |
| pkg_version        | 0.69.1                                          |
| build_time         | 2022-09-28 02:58:08 +00:00                      |
| build_rust_channel | release                                         |
| features           | database, dataframe, default, trash, which, zip |
| installed_plugins  |                                                 |

### Additional context

Running in Windows Terminal on Windows 10 (current version).

_No response_

**Comment by fdncred** [comment]
this is because tabs are evil 🤣 

this may be a bug in miette, the crate we use to create errors. or it could be a bug in our code too.

**Comment by ysthakur** [comment]
Seems to be fixed (screenshot is of version 0.87.1):

![image](https://github.com/nushell/nushell/assets/45539777/5fd70834-e130-445b-9c8f-3c071a0e19a1)

Can you still reproduce the issue @webbedspace?

**Comment by amtoine** [comment]
i can't reproduce either on the latest `main`, thanks @ysthakur :pray: 

i'll close this issues.
@webbedspace, please feel free to reopen if the issue still persists in another instance :wink:
